packages/typedoc-plugin-appium/lib/options/reader.ts

```diff
@@ -104,7 +109,19 @@ export class AppiumPluginOptionsReader implements OptionsReader {
     };
 
     for (const reqdEntryPoint of REQUIRED_PACKAGES) {
-      const foundReqdEP = entryPoints.find((entryPoint) => entryPoint.includes(reqdEntryPoint));
+      const foundReqdEP = entryPoints.find((entryPoint) => {
+        if (hasMagic(entryPoint)) {
+          // this makes it find only dirs
+          entryPoint = entryPoint.replace(/\*$/, '*/');
+          try {
+            const matches = globSync(entryPoint);
+            return matches.includes(reqdEntryPoint);
+          } catch {
+            return false;
+          }
+        }
+        return entryPoint.includes(reqdEntryPoint);
+      });
```

Comment:
now one can use globs for entry points in `typedoc.json`.  we need to check if any of those entry points match the required libs (`@appium/base-driver` and `@appium/types`).